tests/migrations/test_table_engines.py

```diff
@@ -1,23 +1,36 @@
-from snuba.migrations.table_engines import MergeTree, ReplacingMergeTree
+import pytest
 
+from snuba.migrations.table_engines import MergeTree, ReplacingMergeTree, TableEngine
 
-def test_table_engines() -> None:
-    assert (
-        MergeTree(order_by="timestamp",).get_sql() == "MergeTree() ORDER BY timestamp"
-    )
 
-    assert (
-        MergeTree(order_by="date", settings={"index_granularity": "256"}).get_sql()
-        == "MergeTree() ORDER BY date SETTINGS index_granularity=256"
-    )
-
-    assert (
+test_cases = [
+    (
```

Comment:
Could you please use the `pytest.param` helper function that allows you to define an `id` for each test that is then displayed when the test runs ? That makes it much easier to identify specific tests that are failing.
Example: https://github.com/getsentry/snuba/blob/master/tests/query/parser/test_query.py#L145-L158

Reply:
Done